### ISSUE TITLE
Sound isn't downloaded on latest version

### ISSUE THREAD

**Issue by dahmc** [luoliyan/chinese-support-redux#183]
**Describe the bug**
Anki prompted me to install update today for this plugin, so I did (0141)

Since then, sound is not working for newly added cards

When adding a new card, it shows me it has a sound file at the bottom, but nothing ever plays on the card.

Tested on anki 2.1.40, 2.1.41, 2.1.42

**To Reproduce**
Create new card
test it out, sound does not play

**Expected behavior**
Sound should play as usual

**Specs (please complete the following information):**
 - OS: macOS 11.2.3
 - Anki Version 2.1.40, 2.1.41, 2.1.42
- Chinese Support Version 0141

**Comment by karl-an** [comment]
same problem here!

**Comment by dahmc** [comment]
@luoliyan Continuing here as you mentioned.

I just double checked, I'm on 0.14.2, and latest anki. I tried not only in the browser now, but also during reviews.

Here's what I've noticed:
- If I add new words I've not tried before, it works as expected now, nice!
- If I try the words I was testing with like 妳好 it doesn't work. It seems it's cached somewhere and I can't get rid of it.

I've tried cleaning media and all, but to no avail. 

So basically I can now add all my new vocabulary again, which is great, but not sure why the ones that I've tried before will never refresh even though I deleted the cards and cleaned media.

Thanks a lot for all your quick fixes, really appreciate it!

**Comment by luoliyan** [comment]
I expect this will be a relatively easy fix. I'll try my hand at it tonight after work.

Thanks for your help with testing. There are so many cases when something works fine on my system but not on others, so the reports are really valuable.

**Comment by luoliyan** [comment]
Amazon Polly should still work if set up correctly (a bit niche, I guess - but someone else did all the work on that).

As for PRC/Taiwan - these used to be different accents, but if there's truly no difference now, I'll just use PRC and call it "Google".

**Comment by luoliyan** [comment]
Actually, I'll just grey out Amazon Polly if the user hasn't configured it. That should remove some confusion.

**Comment by karl-an** [comment]
> Amazon Polly should still work if set up correctly (a bit niche, I guess - but someone else did all the work on that).
> 
> As for PRC/Taiwan - these used to be different accents, but if there's truly no difference now, I'll just use PRC and call it "Google".

Do you have a link on how to configure it, I would like to give it a try 😊

**Comment by twopools** [comment]
I'm on Anki 2.1.43 with addon v0142 on macOS. I think I have this exact same issue.

All new cards I add don't play sound at all. Old cards still work as expected. Tools -> Check Media... reports all new sound files as missing, always. While adding cards there are no external http connections, I'm sure nothing is fetched.

Since this is marked as fixed and everyone appers to be happy, maybe Anki 2.1.43 broke it again? It sure is not working for me.

Is there any way I can help fixing this?

**Comment by luoliyan** [comment]
@karl-an  I'll see if I can remember how it works. Someone else wrote the code for that, and I tested it only once. If I can get it working for me, I'll add instructions to the readme.
